Fix:

warning: implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)'
--- src/makemsg.c.orig	2006-08-02 13:11:44.000000000 -0500
+++ src/makemsg.c	2020-11-14 09:51:34.000000000 -0600
@@ -19,6 +19,7 @@
  */
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include <unistd.h>
 #ifdef __QNX__
 #include <unix.h>
